packages/server/graphql/public/mutations/startRecurrence.ts

```diff
@@ -54,7 +54,9 @@ const startRecurrence: MutationResolvers['startRecurrence'] = async (
     meetingType: 'teamPrompt',
     title: 'Async Standup',
     recurrenceRule: recurrenceRule.toString(),
-    duration: 24 * 60 // 24 hours
+    duration: 24 * 60, // 24 hours
+    teamId,
+    facilitatorId: viewerId
   })
```

Comment:
I think we need a separate "Meeting Recurrence Started" event here. Would be interesting to see who always setup the cadence for the team, the team lead? billing lead?

Reply:
Added!